libsql-server/src/rpc/replica_proxy.rs

```diff
@@ -38,6 +38,8 @@ impl Proxy for ReplicaProxyService {
         &self,
         req: tonic::Request<tonic::Streaming<ExecReq>>,
     ) -> Result<tonic::Response<Self::StreamExecStream>, tonic::Status> {
+        tracing::debug!("stream_exec");
```

Comment:
I wish you had introduced a separate commit with all those tracing changes that are not really related to metastore.

Reply:
So these were related because they were used to debug test failures that started failing when I added the meta store. 

I will remove them for now since I think these are somewhat low quality logs but in general, I am not sure if there is a large benefit to extracting debug items like this that do not 1) affect the implementation 2) change any production logging.